### DISCUSSION TITLE
Option to create a Series\Author\Books Library along with the current Author\Series\Books Library

### DISCUSSION THREAD

**Discussion by Don-Cal** [advplyr/audiobookshelf-app#1131] [Ideas]
I would really like to keep my book series together rather than separating them to their various authors.

Would it be possible to have the option to create a series Library where the scanner would know to read the folder structure in Series\Author\Books format?

Would it be possible to also merge that folder structure with the current library structure so all books are in the same library?

**Comment by nichwall** [comment]
I think you meant to post this in https://github.com/advplyr/audiobookshelf.

You can disable the folder structure in the library scanner settings, so information is only pulled in from metadata tags or other metadata files and you can name the folders however you want. This does not affect book grouping (they all still need to be in their own folder).

**Reply by Don-Cal** [reply]
Hi Nick,
Thanks for replying.
.... that wasnt what I was suggesting. My metadata is a mess. Its my folder structure that contains the info.

My request is to be able to keep a series of books together with multiple authors. 
Take Star Wars \ Alien v Predator \ Jason Bourne \ etc series which have multiple authors, 
Currently ABS suggests keeping each book with its author so we have AUTHOR\SERIES\XX - BOOK
That means we have to split a series to multiple different authors.

I would like to keep series with different book authors together


Having said that, Im also having the issue of information being pulled from the book files metadata which really messed things up. I spent weeks getting my books in  AUTHOR\SERIES\XX - BOOK format only for them to read nonsense from the book file metadata.

I've since found the option to specify which info takes precedence when importing (ie Folder structure, metadata, text files, etc)
Im intending to delete the audiobookshelf container and metadata>
Reinstall audiobookshelf and configure the import section to take folder data over metadata.
Then reimport the lot.
Once I have it properly set up, I'll then update the metadata in the book files.
